tests/framework/clients/object_user.go

```diff
@@ -48,8 +49,12 @@ func (o *ObjectUserOperation) GetUser(namespace string, store string, userid str
 
 // UserSecretExists Function to check that user secret was created
 func (o *ObjectUserOperation) UserSecretExists(namespace string, store string, userid string) bool {
-	_, err := o.k8sh.GetResource("-n", namespace, "secrets", "-l", "rook_object_store="+store, "-l", "user="+userid)
-	if err == nil {
+	found_sec, err := o.k8sh.GetResource("-n", namespace, "secrets", "-l", "rook_object_store="+store, "-l", "user="+userid)
+	//GetResource(blah) returns success if blah is or is not found.
+	//err = success and found_sec not "No resources found." means it was found
```

Comment:
The code is self explanatory but the comment help explain GetResource()/'kubectl get' returns success even if the resource is not found. I do not know if this is a change in how GetResource()/'kubectl get' works or if the previous usage of this kind of check was lucky. I would like to keep it in case there is a change again. This was a difficult issue to find.